cinder/README.md

```diff
@@ -51,12 +53,13 @@ Create or edit `/home/stack/templates/custom-dellemc-container.yaml`.
 
 ```yaml
```

Comment:
Does OPS16 Cinder support IPv6?

Reply:
Yes, I think so.

Reply:
Why is IPv6 support mentioned in Manila, but not in Cinder? Is there any additional configuration needed for Manila?

Reply:
Yes, there is one more configuration need to set in Manila: 'ManilaIPv6: true'. In Cinder, specify san_ip with ipv6 format is okay.